examples/taxi-ride-akka/akkastreams/src/main/scala/taxiride/akka/processor/actors/singlemessage/RidesBehaviour.scala

```diff
@@ -0,0 +1,47 @@
+package taxiride.akka.processor.actors.singlemessage
+
+import akka.actor.typed._
+import akka.actor.typed.scaladsl.Behaviors
+import taxiride.datamodel.TaxiRideOrFare.MessageType
+import taxiride.datamodel._
+
+import scala.collection.mutable.Map
```

Comment:
You can't just use a mutable map in an object.

Reply:
Actually you can. All of the actors are **_local_**. But doing it cleaner in rework

Reply:
(it's not about it being local, I'm referring to concurrency issues, but good to know it will be reworked)